lib/modules/manager/npm/post-update/pnpm.spec.ts

```diff
@@ -284,6 +285,61 @@ describe('modules/manager/npm/post-update/pnpm', () => {
     ]);
   });
 
+  it('uses volta version and puts it into constraint', async () => {
+    const execSnapshots = mockExecAll();
+    const configTemp = partial<PostUpdateConfig>();
+    const fileContent = codeBlock`
+    {
+  "name": "parent",
+  "version": "1.0.0",
+  "engines": {
+    "pnpm": "=6.15.0"
```

Comment:
```suggestion
    "pnpm": "^6.0.0"
```
this is usually used as range